vitest.config.ts

```diff
@@ -12,7 +12,12 @@ export default defineConfig({
       reporters: process.env.CI ? ['json'] : ['verbose'],
     },
     coverage: {
-      reporter: process.env.CI ? ['lcov'] : ['text', 'json', 'html'],
+      lines: 0,
+      statements: 0,
+      functions: 0,
+      branches: 0,
```

Comment:
whenever tests are run locally this will update to the correct values.  When test coverage drops it will fail the pr unless the author explicitly lowers the test coverage in vitest config.  This is makes it so all test coverage drops are acknowledged and not accidental

Reply:
can you run the tests in this to demo/set a baseline?

Reply:
![image](https://github.com/base-org/op-viem/assets/35039927/12e56bc4-54b6-4f86-a05b-17b5a21c7c21)